<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselMeshingWidget</class>
 <widget class="QWidget" name="VesselMeshingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QFrame" name="solidNameFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="showMeshInformation">
        <property name="toolTip">
         <string>Show mesh statistics</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/VesselMeshing.qrc">
          <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="globalOptionsGroup">
     <property name="title">
      <string>Global options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="editGlobalOptionsButton">
          <property name="toolTip">
           <string>Edit global meshing parameters</string>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="localOptionsGroup">
     <property name="toolTip">
      <string>Select the faces to edit local meshing parameters</string>
     </property>
     <property name="title">
      <string>Local options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Faces</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="selectFacesButton">
          <property name="toolTip">
           <string>Select the faces to apply the local meshing parameters to</string>
          </property>
          <property name="text">
           <string>Select in 3D</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editLocalOptionsButton">
          <property name="toolTip">
           <string>Edit local meshing parameters for selected faces</string>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="faceSettingsTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="meshButton">
       <property name="toolTip">
        <string>Create the mesh</string>
       </property>
       <property name="text">
        <string>Mesh</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/VesselMeshing.qrc">
         <normaloff>:/icons/DataManagerIcon_Mesh.png</normaloff>:/icons/DataManagerIcon_Mesh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancelMeshButton">
       <property name="toolTip">
        <string>Cancel the current mesh operation</string>
       </property>
       <property name="text">
        <string>Cancel mesh</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/VesselMeshing.qrc">
         <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showMeshInformation</tabstop>
  <tabstop>selectFacesButton</tabstop>
  <tabstop>editLocalOptionsButton</tabstop>
  <tabstop>faceSettingsTable</tabstop>
  <tabstop>meshButton</tabstop>
  <tabstop>cancelMeshButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/VesselMeshing.qrc"/>
 </resources>
 <connections/>
</ui>
